LIPs/LIP-25.md

```diff
@@ -0,0 +1,240 @@
+---
+lip: 25
+title: Extensible Governance Contract
+author: Nico Vergauwen (@kyriediculous) <nico@livepeer.org>
+type: Standard
+status: Draft
+created: 2020-06-09
+requires: 30
```

Comment:
Are you planning on creating a separate LIP for https://github.com/livepeer/LIPs/issues/30 or is that issue just a discussion thread for parameters that will be defined in this LIP? If you plan on doing the former then a separate draft LIP would need to be checked in so that it can be properly referenced by this document. If you plan on doing the latter then we don't need the `requires` field here. I suggest the latter as it is a bit easier.

Reply:
Okay let's go with the latter.

Reply:
fixed in ddb128b